airbyte-integrations/connectors/source-monday/source_monday/extractor.py

```diff
@@ -41,9 +41,8 @@ def extract_records(self, response: requests.Response) -> List[Record]:
             return result
 
         for board_data in response_body["data"]["boards"]:
-            if not isinstance(board_data, dict):
+            if not isinstance(board_data, dict) or not board_data.get("activity_logs"):
```

Comment:
Should there be a unit test for the case when `response_body["data"]["boards"]` doesn't have the `activity_logs` section?

Reply:
added